chore: document new MFA enforcement on organization option (#35564)
supabase/supabase: apps/docs/content/guides/platform/org-mfa-enforcement.mdx

Reviewer: Was this intended to be semi-hidden or did you want to add to [Navigation Menu](https://github.com/supabase/supabase/blob/master/apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts)?

Author: No, I forgot about updating the nav 🙏🏼

---
title: 'Enforce MFA on Organization'
description: 'All users in an organization must have a valid MFA session to interact with organization resources'
---

> Supabase provides multi-factor authentication (MFA) enforcement on the organization level. Ensuring all organization members must use MFA and have a valid MFA backed session when interacting with the organization and projects belonging to the organization.

Reviewer: ```suggestion
Supabase provides multi-factor authentication (MFA) enforcement on the organization level. With MFA enforcement, you can ensure that all organization members use MFA. Members cannot interact with your organization or your organization's projects without a valid MFA-backed session.
```

<Admonition type="note">

MFA enforcement is only available on the [Pro, Team and Enterprise plans](https://supabase.com/pricing).

</Admonition>

> 
> When you first create an account, a default organization is created for you and you'll be assigned as the **Owner**. Any organizations you create will assign you as **Owner** as well.

Reviewer: ```suggestion
```

Any particular reason this is here? Seems like a bit of a non-sequitur

Author: 🤦🏼‍♂️ 😅  copy-paste giving me away
